### ISSUE TITLE
SC2221 "This pattern always overrides a later one" should show the patterns

### ISSUE THREAD

**Issue by bje-** [koalaman/shellcheck#1039]
#### For bugs
- Rule Id SC2221 and SC2222
- shellcheck version: 0.4.6
- I tried on shellcheck.net and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [X] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

foo=123  # dummy
case $foo in
  foo) echo 1 ;;
  # many lines of cases
  foo) echo 2 ;;
esac
```

#### Here's what shellcheck currently says:
 
Line 5:
  foo) echo 1 ;;
  ^-- SC2221: This pattern always overrides a later one.
 
Line 7:
  foo) echo 2 ;;
  ^-- SC2222: This pattern never matches because of a previous pattern.

#### Here's what I wanted or expected to see:

It would be great if the duplicated pattern could be shown (and, perhaps the line number). eg:

Line 5:
  foo) echo 1 ;;
  ^-- SC2221: This pattern always overrides a later one (foo on line 7).
 
Line 7:
  foo*) echo 2 ;;
  ^-- SC2222: This pattern never matches because of a previous pattern (foo on line 5).
